nuxeo-features/nuxeo-elasticsearch/nuxeo-elasticsearch-core/src/main/java/org/nuxeo/elasticsearch/client/ESRestClientFactory.java

```diff
@@ -152,9 +150,11 @@ protected SSLContext getSslContext(ElasticSearchClientConfig config) {
         try {
             Path keyStorePath = Paths.get(config.getOption(KEYSTORE_PATH_OPT));
             String keyStorePass = config.getOption(KEYSTORE_PASSWORD_OPT);
-            KeyStore keyStore = KeyStore.getInstance(KeyStore.getDefaultType());
+            String keyStoreType = config.getOption(KEYSTORE_TYPE, KeyStore.getDefaultType());
+            char[] keyPass = StringUtils.isEmpty(keyStorePass)?null:keyStorePass.toCharArray();
```

Comment:
Formatting: add spaces around `?` and `:`
For a user-supplied parameter like this one, you can also use `isBlank` instead of `isEmpty`.